torch/testing/_internal/common_nn.py

```diff
@@ -99,6 +99,15 @@ def get_weight(m):
 
 
 module_tests = [
+    dict(
```

Comment:
Looks good! FYI (I probably need to update the module lab) this list of dicts is being superseded by a database of ModuleInfo entries for generating module tests (akin to OpInfo if you're familiar with that).

https://github.com/pytorch/pytorch/blob/e011a8e18bf469a6a612fd1e7647159c353730a9/torch/testing/_internal/common_modules.py#L994